var/spack/repos/builtin/packages/mercury/package.py

```diff
@@ -67,7 +69,6 @@ def cmake_args(self):
             '-DBUILD_TESTING:BOOL=%s' % str(self.run_tests),
             '-DMERCURY_ENABLE_PARALLEL_TESTING:BOOL=%s' % str(parallel_tests),
             '-DMERCURY_USE_BOOST_PP:BOOL=ON',
-            '-DMERCURY_USE_CHECKSUMS:BOOL=ON',
```

Comment:
instead of adding the following lines, you can simply do:
`'-DMERCURY_USE_CHECKSUMS:BOOL=%s' % variant_bool('+checksum'),`

Reply:
Ah, good call. I just pushed an update that incorporates that change.